### ISSUE TITLE
Issue ID: DAC_Keyboard_Trap1

### ISSUE THREAD

**Issue by VDuczekW** [guardian/dotcom-rendering#5053]
Report Link: https://drive.google.com/file/d/1dcne25k6vqUzC7RMjpV5f_UzvABWKE50/view?usp=sharing

Full issue list: https://docs.google.com/spreadsheets/d/1vWQtZfD49uq7uNCNELuYN_HcIj-BKiE4t4Hwda3tJaQ/edit#gid=0

**Comment by bryophyta** [comment]
>**Note**
>This is an automatically-generated comment. Feel free to edit the content, but please don't edit this 'note' block.
 
# Intro to report

Copying the intro here to aid discoverability of the report's content:
```
Keyboard Trap
Users keyboard focus is trapped within an element.
WCAG Reference:
2.1.2 No Keyboard Trap (Level A)
Understanding No Keyboard Trap | How to Meet No Keyboard Trap
Issue ID: DAC_Keyboard_Trap1
URL: https://www.theguardian.com/crosswords/quick/16138
Page title: Quick crossword No 16,138 | Crosswords | The Guardian
Journey: 1.30
Screenshot:

When traversing the page using the tab key, the users focus seems to be trapped within an
element and there is no appropriate method to escaping this trap, apart from refreshing the
page which is not expected behaviour. After the user’s keyboard focus moves from the
comment button, it is taken to a hidden input element where they cannot remove their
keyboard focus from.
```
